addons/mail/static/src/components/message_reaction_group_summary_view/message_reaction_group_summary_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.MessageReactionGroupSummaryView" owl="1">
+        <div class="o_MessageReactionGroupSummaryView mr-1 mb-1"
+             t-att-class="{ 'o-activeReaction': messageReactionGroup and messageReactionGroup.message.messageViews[0].threadView.messageReactionsSummaryView and messageReactionGroup.content === messageReactionGroup.message.messageViews[0].threadView.messageReactionsSummaryView.highlightedReaction.content }"
```

Comment:
Same remark as before, it should receive prop local id and make getter of a view model (e.g. `MessageReactionSummaryView`) instead of a pure logic model (e.g. `MessageReactionGroup`).

Template is overly complicated because of this. If you had `messageReactionsSummaryView` in prop local id and getter, the template would be much much simpler.